app/controllers/kms/application_controller.rb

```diff
@@ -1,11 +1,13 @@
 module Kms
   class ApplicationController < ActionController::Base
+    layout 'kms/devise'
```

Comment:
that was a reason why I add this layout like you point it out to engine configuration  file
<img width="519" alt="screen shot 2017-01-22 at 12 27 15" src="https://cloud.githubusercontent.com/assets/342814/22182505/2f6edb74-e09e-11e6-8f3d-1fa98847f92f.png">